components/donation/Donation.component.tsx

```diff
@@ -61,7 +61,15 @@ export default function DonationView({ nodeInfo, show }: ParentProps) {
 
   const resp = useSWR("/api/offer", fetcher);
 
-  let bolt12: string = resp.data != undefined ? resp.data.offer.bolt12 : null;
+  let bolt12: string =
+    resp.data != undefined ? resp.data.offer[0].bolt12 : null;
+
+  const commandFetchInvoice =
+    "fetchinvoice " +
+    bolt12 +
+    " [msatoshi] [quantity] [recurrence_counter] [recurrence_start] [recurrence_label] [timeout] [payer_note] \n";
```

Comment:
`  const commandFetchInvoice = `lightning-cli fetchinvoice ${bolt12}` \n";`

The command needs to be pasted in the command line if you ran the actual comment core lightning will return an error